src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public User findByEmail(String email, String password) {
```

Comment:
```suggestion
    public User findByEmail(String emai) {
```
you cannot change the method signature